draft-foudil-securitytxt.md

```diff
@@ -421,8 +421,8 @@ sign-header      =  <headers and line from section 7 of [RFC4880]>
 
 sign-footer      =  <OpenPGP signature from section 7 of [RFC4880]>
 
-unsigned         =  *line [canonical-field eol *line] [lang-field eol] *line
-unsigned         =/ *line [lang-field eol *line] [canonical-field eol] *line
+unsigned         =  *line (can-field eol) *line contact-field *line (lang-field eol) *line
```

Comment:
I like this! But it gets rid of the square brackets, which were made the Canonical and Preferred-Languages directives optional. This can be remedied by replacing the parentheses.

Another issue is that I think contact-field needs to have an "eol" suffixed to it.

```suggestion
unsigned         =  *line [can-field eol] *line (contact-field eol) *line [lang-field eol] *line
```

I also have a question: if order doesn't matter, then two `*line`s can appear next to each other. If there are three immaterial lines then that can be distributed as (0,3); or (1,2); or (2,1); or (3,0). There is more than one way to parse a security.txt file, making the grammar ambiguous. I'm not sure if this really matters though, given that a human programmer is going to have to transcribe the ABNF comment into a programming language or write out a lot of BNF, so they will likely be able to prevent any ambiguous parsing.

If we did want to avoid it, though, I think it would be fairly easy: just move the *lines into parentheses so that they have to be in front of a material directive. We'd then need a *line at the end. This might make things a tad more confusing so it depends on whether it's worth it.

A stylistic question: do you think it looks prettier with a space after the semicolon (the one which denotes the start of an ABNF comment?

Reply:
You are correct regarding "[]" vs "()", I am fixing that

re: *line - I think we are relaying on a human programmer anyway because of the order comment, so I would rather leave it the way it is

Regarding the style, space makes sense - I will add it

Thank you again